app-modules/prospect/tests/Prospect/ListProspectTest.php

```diff
@@ -133,3 +136,31 @@
                 ->status_id->toBe($status->id)
         );
 });
+
+it('can filter prospects by `subscribed` prospects', function () {
+    $user = User::factory()->licensed(LicenseType::cases())->create();
+
+    $user->givePermissionTo('prospect.view-any');
+    $user->givePermissionTo('subscription.view-any');
+    $user->givePermissionTo('subscription.*.view');
+
+    actingAs($user);
+
+    $subscribedProspects = Prospect::factory()->count(3)->hasSubscriptions()->create();
```

Comment:
What is the purpose of this group of Prospects here? And why does it have the same variable name as the one below it?